cfgov/v1/jinja2/v1/includes/organisms/post-preview.html

```diff
@@ -74,7 +74,7 @@
                     {% if loop.index > 1 %}
                         <span class="m-meta-header_separator">|</span>
                     {% endif %}
-                    <span class="m-meta-header_category">{{ category_slug.render(category=cat.name, href=url) }}</span>
+                    <span class="m-meta-header_category">{{ category_slug.render(category=cat.name) }}</span>
```

Comment:
@chosak This "meta header" is only found in post previews and it doesn't appear they are ever links. Is there any other cleanup that can happen here?

Reply:
@anselmbradford the `href` parameter is currently used in the PostPreviewSnapshot module, in the template [here](https://github.com/cfpb/consumerfinance.gov/blob/e19568831dab1f29f417cd6545ab522ce0981e37/cfgov/v1/jinja2/v1/includes/organisms/post-preview-snapshot.html#L28).